pkg/reconciler/knativeserving/ingress/ingress_test.go

```diff
@@ -78,8 +77,9 @@ func TestAppendInstalledIngresses(t *testing.T) {
 	tests := []struct {
 		name                 string
 		instance             servingv1alpha1.KnativeServing
-		expected             bool
+		expectedNoErr        bool
 		expectedResourcesNum int
```

Comment:
`expectedResourcesNum` is not used. Let's remove it.

Reply:
will do